deployments/ansible/README.md

```diff
@@ -8,6 +8,24 @@ The role installs Splunk OpenTelemetry Connector configured to
 collect metrics, traces and logs from Linux machines and send data to [Splunk 
 Observability Cloud](https://www.splunk.com/en_us/observability.html). 
 
+## Linux
+Currently, the following Linux distributions and versions are supported:
+
+- Amazon Linux
+- RedHat
+- Debian: buster, stretch
+- Ubuntu: 16.04, 18.04, 20.04
+
+## Windows
+Currently, the following Windows versions are supported:
+Ansible requires PowerShell 3.0 or newer and atleast .NET4.0 to be installed on Windows host.
+A WinRM listner should be created and activeted. 
+For setting up Windows Host Refer:[Ansible Docs](https://docs.ansible.com/ansible/latest/user_guide/windows_setup.html).
+
+- Windows Server 2012 64-bit
+- Windows Server 2016 64-bit
+- Windows Server 2019 64-bit
+
```

Comment:
No need to duplicate this info here and in the role's README. Let's remove it from here